specs/WebResourceRequested-CustomScheme.md

```diff
@@ -70,12 +70,14 @@ options.CustomSchemeRegistrations.Add(
   new CoreWebView2CustomSchemeRegistration(customScheme)
   {
     TreatAsSecure = true,
-    AllowedOrigins = { "https://*.example.com" }
+    AllowedOrigins = { "https://*.example.com" },
+    HasAuthorityComponent = true
```

Comment:
Saying 'has authority' for this doesn't seem right to me. Its not whether there's an authority its how we produce an origin from the URI right? If it were just whether or not there's an authority then we wouldn't need a separate property for this we could just parse the URI. Is this whether or not it has an opaque origin (defined [in that HTML link](https://html.spec.whatwg.org/multipage/origin.html))?